src/wms/src/main/java/org/geoserver/wms/capabilities/DimensionHelper.java

```diff
@@ -264,7 +264,22 @@ void handleRasterLayerDimensions(final LayerInfo layer) throws RuntimeException,
     private void handleElevationDimensionRaster(
             CoverageInfo cvInfo, DimensionInfo elevInfo, ReaderDimensionsAccessor dimensions)
             throws IOException {
-        TreeSet<Object> elevations = dimensions.getElevationDomain();
+        TreeSet<Object> elevations = null;
+        try {
+            if (elevInfo.getPresentation() != DimensionPresentation.LIST) {
+                Double minValue = dimensions.getMinElevation();
+                if (minValue != null) {
+                    elevations = new TreeSet<>();
+                    elevations.add(minValue);
+                    elevations.add(dimensions.getMaxElevation());
+                }
+            }
+            if (elevations == null) {
+                throw new Exception();
```

Comment:
The exception should provide some reason, as it goes up and becomes the OGC ServiceException response text